src/gui/guiPasswordChange.cpp

```diff
@@ -66,11 +63,8 @@ void GUIPasswordChange::regenerateGui(v2u32 screensize)
 	/*
 		Calculate new sizes and positions
 	*/
-#ifdef HAVE_TOUCHSCREENGUI
 	const float s = m_gui_scale * RenderingEngine::getDisplayDensity() / 2;
```

Comment:
There are two problems with this:

1. It's a weird hack, `m_gui_scale` already takes the display density into account.

2. It causes the "change password" dialog to become unusably small for some display density values.

```suggestion
	const float s = m_gui_scale;
```